deploy/gke/simple/README.md

```diff
@@ -67,6 +67,171 @@ Since there is no application currently running on the cluster, the collector is
 
+## Try it out yourself (Optional)
+
+Thus far, we've set up an OpenTelemetry collector configured to scrape itself and send metrics to the configured exporters. In this section we will introduce external source(s) of telemetry data to see how the collector operates on these sources. The sources will include data representing metrics, logs and traces.
+
+You should be able to follow all steps mentioned in this README till [Verify the Deployment](#verify-the-deployment) successfully. This ensures that there are no permissions issues and you are able to succeesfully run a GKE cluster and connect to it.  
+
+### Source of telemetry data
+
+The example uses JSON file(s) containing telemetry data in OTLP format as source of telemetry data. These files can contain metrics, traces or logs - but each file should contain only a single type of telemetry data. Sample files have been provided in the folder - [otlp-data](./otlp-data/). The collector running in the cluster will then read these files and treat the data in them as if it were coming from a running application. 
+
+### Updating the collector configuration to receive data from JSON file
+
+We need to update the collector configuration file - [otel-config](./otel-config.yaml) to add a reciever that is able to receive telemetry data from the added JSON file.
+
+Under the `receivers` section in the config file, add the following configuration - 
+
+```
+  otlpjsonfile:
+    start_at: "beginning"
+    include:
+      - "/mnt/testdata/metrics/*.json"
+      - "/mnt/testdata/traces/*.json"
+      - "/mnt/testdata/logs/*.json"
+```
+For more details about this particular receiver, check [otlpjsonfilereceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/otlpjsonfilereceiver).
+
+***NOTE:** The path in `include` points to where the file would be mounted within the Kubernetes cluster environment and is therefore different from where it is present on your local machine. More information on this in [the next section](#making-the-json-file-available-within-the-cluster).*
+
+Next, add this receiver in the traces pipeline, so your trace pipeline looks like - 
+
+```
+  traces:
+    receivers: [otlp, otlpjsonfile]
+    processors: [memory_limiter, batch, resourcedetection/gke]
+    exporters: [googlecloud, logging]
+
+  metrics:
+    receivers: [otlp, prometheus/self, otlpjsonfile]
+    processors: [memory_limiter, batch, resourcedetection/gke]
+    exporters: [googlecloud, logging]
+  
+  logs:
+    receivers: [otlp, otlpjsonfile]
+    processors: [memory_limiter, batch, resourcedetection/gke]
+    exporters: [googlecloud, logging]
+```
+
+For exporting logs to Google Cloud Platform, we need to further configure the collector with the log name and the GCP project ID. The project ID is used by the cloud exporter to create GCP log entries. 
```

Comment:
is project_id not inferred by the collector based on the GKE node it's running on?

Reply:
No, it was not since the logs were not being generated. I can double check again though by removing it and keeping the `default_log_name`.

Reply:
Just double checked, turns out that `default_log_name` was the required field in this case. Project ID is inferred.